src/panels/config/cloud/login/cloud-login.js

```diff
@@ -147,13 +146,13 @@ class CloudLogin extends LocalizeMixin(
                   progress="[[_requestInProgress]]"
                   >[[localize('ui.panel.config.cloud.login.sign_in')]]</ha-progress-button
                 >
-                <button
+                <mwc-button
```

Comment:
I think was styled as a link to not stand out too much

Reply:
Then I think it has the opposite effect (at least to me). Since it looks different than all other buttons, it stands out 😆 .

Reply:
Can you add a screenshot?

Reply:
![image](https://user-images.githubusercontent.com/114137/101049403-c5c5e200-3583-11eb-83af-711cd8ee2230.png)

Reply:
Now it has the same importance as the sign in button, that's not good... I also feel like the login button should be on the right side?

Maybe move the forgot password under the password input field?

Reply:
I agree with the comments by @SeanPM5. The "Sign In" button to me makes the most sense on the left, since that is the primary action and you are drawn to the left side to the input fields already. For other similar cards (e.g. password change card) we also have that button on the left.

The "Forgot Password?" is quite far away so I do not think it is prominent. Moving that below the input fields has three negative effects IMHO:
1. Wastes screen space since the card gets higher, whereas in the action row we have space anyway on the right side.
2. Makes the whole card feel very left heavy, despite being rather wide.
3. Hides the password recovery option visually (I get it should not be too prominent, but as I said I don't think it is this way).